assets/tests/components/mapPage.test.tsx

```diff
@@ -133,14 +129,20 @@ jest.mock("../../src/hooks/useAllStops", () => ({
   useAllStops: jest.fn(() => []),
 }))
 
-jest.mock("../../src/tilesetUrls", () => ({
+jest.mock("../../src/hooks/useStations", () => ({
   __esModule: true,
-  tilesetUrlForType: jest.fn(() => null),
+  useStations: jest.fn(() => []),
+}))
+
+jest.mock("../../src/hooks/useMinischedule", () => ({
+  __esModule: true,
+  useMinischeduleRun: jest.fn(),
+  useMinischeduleBlock: jest.fn(),
 }))
 
-jest.mock("userTestGroups", () => ({
+jest.mock("../../src/tilesetUrls", () => ({
   __esModule: true,
-  default: jest.fn(() => []),
+  tilesetUrlForType: jest.fn(() => null),
 }))
```

Comment:
Okay, after a bit of investigation I've come to the conclusion that:
1. We can remove specific mocks that are just a call to `jest.fn()` without providing a specific implementation.
2. We can mostly remove the `__esModule: true` stuff, _except_ for mocking modules with a default export that we need to mock.
For the examples where we do provide a mock implementation, even if it just returns `null` or `[]`, I think it's better to leave those so that we aren't instead getting an `undefined` value and possibly changing test behavior.

Reply:
> For the examples where we do provide a mock implementation, even if it just returns null or [], I think it's better to leave those so that we aren't instead getting an undefined value and possibly changing test behavior.

I agree with this, my concern is that once we do the [ticket to start resetting mocks after every test](https://app.asana.com/0/1148853526253420/1203482629607208), as I understand it, these default implementation's will be reset to returning `undefined`.
I'm fine with leaving that work till the ticket, but I do think it sort-of hides each test's mock dependencies.

---

aside: I've been mulling over the idea that we provide [manual mocks](https://jestjs.io/docs/manual-mocks) for stuff like this where we expect it to return something valid (e.g., an empty array, or `null`)

But I'm concerned that that solution also hides the mock dependencies as mentioned above.

---

Anyway, thanks for looking into this!

Reply:
Yeah, agreed that there's more work to be done here. I think / hope that for many components there will be some baseline set of mocks that it always needs in order to render, which can then be tweaked for individual tests, so we could probably put those in a helper function that's explicitly invoked by each test, giving us a good balance of being explicit about mocks while not repeating too much code.